frontend/src/static/js/v_authorship.js

```diff
@@ -245,22 +246,32 @@ window.vAuthorship = {
             ? this.getContributionFromAllAuthors(contributionMap)
             : contributionMap[this.info.author];
 
-        if (lineCnt) {
-          const out = {};
-          out.path = file.path;
-          out.lineCount = lineCnt;
-          out.active = lineCnt <= COLLAPSED_VIEW_LINE_COUNT_THRESHOLD;
-          out.wasCodeLoaded = lineCnt <= COLLAPSED_VIEW_LINE_COUNT_THRESHOLD;
-          out.fileType = file.fileType;
+        // skip file not touched by author
+        if (!this.info.isMergeGroup && !(this.info.author in contributionMap)) {
+          return;
+        }
+
+        if (this.info.isMergeGroup && lineCnt === 0) {
+          return;
+        }
+
```

Comment:
And here also looks like legacy code. Could you please help go through the code once more and remove all these sections?

Also if possible can help squash your own changes into one/a few commits so it is easier for us to just look at the new changes.

Reply:
This does seem to cause bug where the report does not show binary files when the group is merged. I have removed this section. So far I did not find other such sections, but please let me know if there is